modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-a-helm-chart.adoc

```diff
@@ -25,3 +28,28 @@ $ helm upgrade --set global.serverStrategy=__<single-host>__
 ----
 
 Depending on the strategy used, replace the `_<single-host>_` option in the above example with `multi-host` or `default-host`.
+
+
+Example for `gateway` single-host strategy type:
+
+[subs="+quotes"]
+----
+$ helm install --namespace __<namespace>__ --set global.serverStrategy=single-host --set global.singleHostExposure=gateway --set cheSinglehostGateway.deploy=true
+----
+
+or:
+
+[subs="+quotes"]
+----
+$ helm upgrade --install che --namespace __<namespace>__ --set global.serverStrategy=single-host --set global.singleHostExposure=gateway --set cheSinglehostGateway.deploy=true
+----
+
+Alternatively, predefined values file can be used as follows:
+
+[subs="+quotes"]
+----
+$ helm upgrade --install che --namespace __<namespace>__ \
+...
+-f "${SOURCE_HOME_DIR}"/deploy/kubernetes/helm/che/values/single-host-gateway.yaml \
```

Comment:
```suggestion
-f "${CHE_SOURCE_DIR}"/deploy/kubernetes/helm/che/values/single-host-gateway.yaml \
```
but I'm not sure with this. I found only one case when referring to template in sources here https://www.eclipse.org/che/docs/che-7/administration-guide/authenticating-users/#_changing_the_authentication_mode and it's not saying anything to where the file is. So this is probably better. @themr0c @boczkowska are there any guidelines/best practice how to reference the Che source-code?